src/Core/Command/UploadAvatarHandler.php

```diff
@@ -96,7 +96,10 @@ public function handle(UploadAvatar $command)
         $this->validator->assertValid(['avatar' => $file]);
 
         $manager = new ImageManager;
-        $manager->make($tmpFile)->fit(100, 100)->save();
+
+        $encodedImage = $manager->make($tmpFile)->fit(100, 100)->encode('jpg', 100);
+
+        @file_put_contents($tmpFile, $encodedImage);
```

Comment:
It doesn't work. That's why we have to use the native `file_put_contents` to save the encoded image.